src/backend/templates/schedule.hbs

```diff
@@ -539,9 +542,18 @@
                     $(sessionName).html(textArray.join(' ') + '...');
                 }
               })
+              roomCount++;
             });
+
             if(width < $(rooms).width()) {
-              $(rooms).width(width);
+              offset = ($(rooms).width() - width)/roomCount;
```

Comment:
Also carrying the discussion further here https://github.com/fossasia/open-event-webapp/issues/1256